packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -641,10 +638,16 @@ Information about project "Runner":
   testWithoutContext('scheme for default project is Runner', () {
     final XcodeProjectInfo info = XcodeProjectInfo(<String>['Runner'], <String>['Debug', 'Release'], <String>['Runner'], logger);
 
-    expect(info.schemeFor(BuildInfo.debug), 'Runner');
-    expect(info.schemeFor(BuildInfo.profile), 'Runner');
-    expect(info.schemeFor(BuildInfo.release), 'Runner');
-    expect(info.schemeFor(const BuildInfo(BuildMode.debug, 'unknown', treeShakeIcons: false)), isNull);
+    expect(info.schemeFor('Runner', flavor: BuildInfo.debug.flavor), 'Runner');
+    expect(info.schemeFor('Runner', flavor: BuildInfo.profile.flavor), 'Runner');
+    expect(info.schemeFor('Runner', flavor: BuildInfo.release.flavor), 'Runner');
+    expect(info.schemeFor('Runner', flavor: const BuildInfo(BuildMode.debug, 'unknown', treeShakeIcons: false).flavor), isNull);
+  });
+
+  testWithoutContext('scheme for a custom project with flavor is still flavor', () {
+    final XcodeProjectInfo info = XcodeProjectInfo(<String>['Runner'], <String>['Debug', 'Release'], <String>['TestFlavor', 'Runner'], logger);
```

Comment:
This line is getting log, suggest adding a comma after `logger` and putting each on its own line.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-a-trailing-comma-for-arguments-parameters-and-list-items-but-only-if-they-each-have-their-own-line